wiki/Building-From-Source.md

```diff
@@ -70,7 +70,7 @@ If you want to edit and rebuild the code, you can skip the `prepare-deps` comman
 
 ## Fedora CUDA installation
 
-If you are here for CUDA installation on Fedora you're at the right place! Else continue down to [Client Building](https://github.com/alvr-org/ALVR/wiki/Building-From-Source#client-building)
+If you are here for CUDA installation on Fedora you're at the right place! Else continue down to [App Building](https://github.com/alvr-org/ALVR/wiki/Building-From-Source#app-building)
```

Comment:
This document is aimed at developers, we don't *have* to do the renaming. But if you think it's still better, then you can use "Android App Building", to be clearer.

Reply:
I agree that it needs Android mention